tests/integration/bond_test.py

```diff
@@ -369,3 +369,11 @@ def test_bond_with_empty_ipv6_static_address(eth1_up):
         assertlib.assert_state(bond_state)
 
     assertlib.assert_absent('bond99')
+
+
+def test_create_vlan_over_bond99_slave_interface(bond99_with_slave):
```

Comment:
No need to mention the `slave` in the test name.

How about: `test_create_vlan_over_a_bond`